app/views/machines/show.html.erb

```diff
@@ -1 +1,14 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+
+<section>
+  <h3>Snacks</h3>
+  <ul>
+    <% @machine.snacks.each do |snack| %>
+      <li>Snack: <%= snack.name %>, Price: $<%= snack.price %></li>
+    <% end %>
+  </ul>
+</section>
+
+<section>
+  <p>Average Price of Snacks: $<%= @machine.average_snack_price %></p>
```

Comment:
to do what I mentioned above, you could do something like `number_to_currency(@ machine.average_snack_price)` and this would return $2.40 or whatever the price was... it would return that dollar sign as well.